.rubocop.yml

```diff
@@ -0,0 +1,41 @@
+AllCops:
```

Comment:
[OPTIONAL]
While this configuration is fine, it is better to use the Ruby On Rails Specific configuration for the curriculum found here: https://github.com/microverseinc/linters-config/tree/master/ror

Reply:
Thank you for the review Maya, will let you know when I get stuck.